backend/game.py

```diff
@@ -41,6 +55,8 @@ def run(self) -> None:
                         self.typing_indicator(c)
                     case OrderComponent() as component:
                         self.manager.send(Message(data=component))
+                    case OrderSubmission() as component:
+                        self.manager.send(Message(data=component))
```

Comment:
`OrderSubmission` is sent by the frontend when the manager submits the order for scoring. also, this is handled in a different pr: https://github.com/Capstone-Projects-2025-Spring/aac-go-fish/pull/119